api/apps/tags/model.coffee

```diff
@@ -26,6 +26,7 @@ Joi.objectId = require('joi-objectid') Joi
   offset: @number()
   count: @boolean().default(false)
   public: @boolean()
+  safe: @boolean().default(false)
```

Comment:
Curious what the naming of `safe` comes from? Is that a common term for this functionality? As a naive person, my guess of what `safe` might do isn't really related to string matching. I think when we discussed I mentioned maybe using `strict`. `exact` could be clearer too? Not a blocker, but I'd like to hear your thoughts. Hardest thing in CS is naming 😸